src/helpers/daily-data-providers/common-mindful-and-therapy.ts

```
@@ -0,0 +1,40 @@
+    });
+
+    return result;
+}
+
+export function combineResults(startDate: Date, endDate: Date, queryResults: DailyDataQueryResult[]): DailyDataQueryResult {
+    const result: DailyDataQueryResult = {};
+
+    let currentDate = startDate;
+    while (currentDate < endDate) {
+        const dayKey = getDayKey(currentDate);
+        queryResults.filter(queryResult => queryResult.hasOwnProperty(dayKey)).forEach(queryResult => {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)